app/cli/cmd/workflow_create.go

```diff
@@ -79,6 +109,10 @@ func newWorkflowCreateCmd() *cobra.Command {
 	cmd.Flags().StringVar(&team, "team", "", "team name")
 	cmd.Flags().StringVar(&contract, "contract", "", "the ID of an existing contract or the path/URL to a contract file. If not provided an empty one will be created.")
 
+	cmd.Flags().BoolVarP(&skipRACreate, "skip-robot-account-create", "s", false, "Skip creating a Robot Account for this workflow.")
+
+	cmd.Flags().SortFlags = false
```

Comment:
I didn't know about this option. Why did you disable it?

Reply:
because if you don't, this is sorted and some options are shown below skip-robot...